mindsdb/integrations/handlers/ray_serve_handler/ray_serve_handler.py

```diff
@@ -42,9 +42,11 @@ def create(self, target: str, df: Optional[pd.DataFrame] = None, args: Optional[
             raise Exception("Error: Training failed: " + resp['status'])
 
     def predict(self, df, args=None):
-        args = self.model_storage.json_get('args')  # override any incoming args for now
+        args = {**(self.model_storage.json_get('args')), **args }  # merge incoming args
```

Comment:
maybe extracting this key and sending to ray server will be enough?
`pred_args = args.get('predict_params', {})`
in pred_args  should be everything from `using` clause of the query:
```
SELECT *
FROM table as t
JOIN model as m
using my_var=12;
``` 
Or something else required to be passed to ray server?

Reply:
I see that there are could be timeseries settings. Maybe is better to pass them in [`create` method](https://github.com/mindsdb/mindsdb/blob/783878df07c50aa6ab5a11ecf322ed3c8baa2d48/mindsdb/integrations/handlers/ray_serve_handler/ray_serve_handler.py#L28) to rayserve model. Then the model can memorize required timeseries parameters